src/HydrationRepository.js

```diff
@@ -1,47 +1,54 @@
 class HydrationRepository {
   constructor(today) {
     this.individualEntryRecords = [];
-    // this.todaysWaterData = this.findTodaysTotalWater(today);
     this.averageOuncesAllTime = this.getAverageOuncesOverall();
   }
   
+  getOuncesByDay(date) {
+    let dayFound = this.individualEntryRecords.find(entry => entry.date === date);
+    return dayFound ? dayFound.numOunces : 0;
+  }
+
   getAverageOuncesOverall() {
     let sumOunces = this.individualEntryRecords.reduce((sum, entry) => {
       sum += entry.numOunces;
       return sum;
     }, 0);
+    let overallAverageOunces = (sumOunces / this.individualEntryRecords.length).toFixed(0)
+    return Number(overallAverageOunces);
+  }
+  
+  getWeeksDailyOunces() {
+    let week = this.individualEntryRecords.slice(-7, -1);
+    return week.map(day => {
+      return day.numOunces;
+    });
     return sumOunces / this.individualEntryRecords.length;
   }
 
-// DOM stuff 
-  // findTodaysTotalWater(todaysDate) {
-  //   return this.individualEntryRecords.find(record => record.date === todaysDate).numOunces;
-  // }
-    // return allTodayWaterEntries.reduce((sum, entry) => {
-    //   sum += Number(entry.numOunces);
-    //   return sum;
-    // }, 0);
-  // }
-
   getOuncesByDay(date) {
     return this.individualEntryRecords.find(entry => entry.date === date);
   }
 
   getWeekAvgOunces() {
-    let week = this.individualEntryRecords.slice(-7);
+    let week = this.getWeeksDailyOunces();
     let numOuncesTotal = week.reduce((sum, entry) => {
-      sum += entry.numOunces;
+      sum += entry;
       return sum;
     }, 0);
-    return Number((numOuncesTotal / 7).toFixed(0));
+    let weekAverageOunces = (numOuncesTotal / 7).toFixed(0);
+    return Number(weekAverageOunces);
   }
 
-  getWeeksDailyOunces() {
-    let week = this.individualEntryRecords.slice(-7)
-    return week.map(day => {
-      return day.numOunces;
-    });
+  addHydroInfo(input) {
```

Comment:
I still think info should be replaced with stats for clarity, thoughts or would this be too much to bother with?